src/EdFi.AnalyticsMiddleTier.DataStandard2/Base/MSSQL/0030-View-SectionDim-for-Student-Engagement.sql

```diff
@@ -0,0 +1,57 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
```

Comment:
![image](https://user-images.githubusercontent.com/9324390/125664224-1b440640-547d-48ef-a228-ed2673086a5a.png)

I was able to get rid of `DISTINCT`  by introducing the join condition for `CourseOffering.TermDescriptorId`.

Reply:
The change to `LocalEducationAgencyId` is now in Slack for discussion.

Reply:
`LocalEducationAgencyId` has now been restored.